examples/src/fibonacci/fib2/prover.rs

```diff
@@ -50,14 +50,16 @@ impl<H: ElementHasher> FibProver<H> {
 
 impl<H: ElementHasher> Prover for FibProver<H>
 where
-    H: ElementHasher<BaseField = BaseElement>,
+    H: ElementHasher<BaseField = BaseElement> + Sync,
```

Comment:
Not for this PR, but I wonder if we should add `Sync` as a required trait for `ElementHasher`.

Reply:
Created issue